session1/main.py

```diff
@@ -0,0 +1,64 @@
+import sys
+import os
+import pygame
+
+"""
+SETUP section - preparing everything before the main loop runs
+"""
+pygame.init()
+
+# Global constants
+SCREEN_WIDTH = 1000
+SCREEN_HEIGHT = 800
+FRAME_RATE = 60
+
+# Useful colors 
+BLACK = (0, 0, 0)
+WHITE = (255, 255, 255)
+
+# Creating the screen and the clock
+screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+clock = pygame.time.Clock()
+
+# Create our 50 x 100 rectangle surface at x = 0, y = 0
+rect = pygame.Rect(0, 0, 50, 100)
+rect_speed_x = 7
+rect_speed_y = 5
+
+while True:
+    """
+    EVENTS section - how the code reacts when users do things
+    """
+    for event in pygame.event.get():
+        if event.type == pygame.QUIT: # When user clicks the 'x' on the window, close our game
```

Comment:
PEP8 says to double-space
```suggestion
        if event.type == pygame.QUIT:  # When user clicks the 'x' on the window, close our game
```